lucene/suggest/src/java/org/apache/lucene/search/suggest/document/NRTSuggesterBuilder.java

```diff
@@ -43,6 +44,9 @@
    * in the output
    */
   public static final int PAYLOAD_SEP = '\u001F';
+  static {
+    assert PAYLOAD_SEP == ConcatenateGraphFilter.SEP_CHAR;
```

Comment:
This can ensured in a dedicated test ?

Reply:
I think it's useful to read the code and both see what's it's value is and that it's equal to something else without having to navigate or use IDE functions.  But this is no big deal to me; I can put back.  Presumably if these values were not the same then some a test would fail but I'm not sure.